Src/BootCamp.Chapter/Subject.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Subject
```

Comment:
Instead, since it doesn't do anything- maybe it could be just an interface? These kind of interfaces have a special name- Maker interface.